tests/test_decorator.py

```diff
@@ -354,3 +354,55 @@ def fn_with_defaults(selenium, a=a, b=b, *, c=c, d=d):
         return [a, b, c, d]
 
     assert fn_with_defaults(selenium) == [a, b, c, d]
+
+
+def test_vararg(selenium):
+    @run_in_pyodide
+    def fn_with_vararg(selenium, a, *args: Callable):
```

Comment:
Why does args has a type annotation 'Callable'?

Reply:
It's a test to make sure that type annotations don't cause errors. The types aren't needed to be correct.